src/HotChocolate/Core/src/Types.Scalars/ScalarResources.resx

```diff
@@ -60,6 +60,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="EmailAddress_Description" xml:space="preserve">
+    <value>The EmailAddress scalar type represents a field whose value conforms to the standard internet email address format as specified in HTML Spec</value>
+  </data>
+  <data name="EmailAddress_IsEmpty_ParseLiteral" xml:space="preserve">
```

Comment:
The Email address is not really empty if the validation failed. It;s more likely to be some error in the email address that makes it invalid :) 

So i suggest to change the name from `IsEmpty` to `IsInvalid`